### DISCUSSION TITLE
Are we going to release Loop and Merge function anytime soon?

### DISCUSSION THREAD

**Discussion by victorx98** [langgenius/dify#4595] [Suggestion]
I know we are working on it "Logic Processing：IF/ELSE, Merge (coming soon), Loop (coming soon)"
Just being curious whether we have exact time line?
They are very useful functions to many applications.

Thanks

**Comment by duanckham** [comment]
Still no response.

**Reply by patryk20120** [reply]
Loops have been deployed and it is called `Iteration` blocks.

If it comes to "merge" you can use `Variable Aggregator` block.
